packages/css/src/components/radio/radio.scss

```diff
@@ -44,111 +35,158 @@
   gap: var(--ams-radio-gap);
   line-height: var(--ams-radio-line-height);
   outline-offset: var(--ams-radio-outline-offset);
+  position: relative;
 
-  &:hover {
-    color: var(--ams-radio-hover-color);
-    text-decoration-line: underline;
-    text-decoration-thickness: var(--ams-radio-hover-text-decoration-thickness);
-    text-underline-offset: 0.375rem;
+  // Radio button check mark
+  &::after {
+    background-color: var(--ams-radio-button-checked-background-color);
+    block-size: calc(var(--ams-radio-font-size) * var(--ams-radio-line-height));
+    flex-shrink: 0;
+    inline-size: 1.5rem;
```

Comment:
Where does this static value come from? I would expect a token and something related to the font-size/line-height perhaps. I could be wrong, it looks good.

Reply:
The radio button is currently a fixed size for all screen widths in the Fabrique designs, it's always 1.5rem. We haven't handled radios on smaller screens in our own designs yet.

Good idea to turn it into a token!

Reply:
Ok, so this will not scale like the font?

Reply:
Currently, no. Unless the design changes.